OpenRA.Game/Platform.cs

```diff
@@ -193,6 +203,13 @@ public static string ResolvePath(string path)
 			if (IsPathRelativeToSupportDirectory(path))
 				path = SupportDir + path.Substring(1);
 
+			// Paths starting with . are relative to the game dir
+			if (path == "?")
```

Comment:
As discussed in IRC, IMO we should instead drop all of these magic characters and instead expose a default set of explicit mounts (maybe prefix them with `^`, so the `Packages` list might have entries like `^SupportDir|Content/cnc`, `^EngineDir|/mods/common/: common`, etc).

We should do this in its own PR that we merge first. Do you want to take this on, or should I?

Reply:
I'm quite busy right now, so would be great if you could do it.